phone-book/src/contacts.js

```diff
@@ -1,39 +1,73 @@
-
-
 class Contacts {
-  constructor(containerSelector) {
-    this.container = document.body.querySelector(containerSelector);
+  constructor(app) {
+    this.title = "Contacts";
+    this.app = app;
+    this.appContainer = app.appContainer;
```

Comment:
Contacts should not know anything about app.appContainer it would make the whole architecture bound and much harder to refactor